src/main/kotlin/no/nav/familie/klage/kabal/domain/KlageinstansResultat.kt

```diff
@@ -19,6 +20,8 @@ class KlageinstansResultat(
     val kildereferanse: UUID,
     val journalpostReferanser: StringListWrapper,
     val behandlingId: UUID,
+    @Column("arsak_feilregistrert")
+    val årsakFeilregistrert: String? = null,
```

Comment:
Må kanskje ha et migreringsscript for å legge til feltet i databasen?

Reply:
Det hjelper nok 🙃